aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -48,29 +48,40 @@ The following minimum changes are required to set up an existing ASP.NET Core pr
 
      ::: moniker-end
 
-1. Publish the app to a folder. Use [dotnet publish](/dotnet/articles/core/tools/dotnet-publish) or a [Visual Studio publish profile](xref:host-and-deploy/visual-studio-publish-profiles) that publishes to a folder.
+1. Publish the app. Use [dotnet publish](/dotnet/articles/core/tools/dotnet-publish) or a [Visual Studio publish profile](xref:host-and-deploy/visual-studio-publish-profiles).
 
    To publish the sample app from the command line, run the following command in a console window from the project folder:
 
    ```console
-   dotnet publish --configuration Release --output c:\svc
+   dotnet publish --configuration Release
    ```
 
-1. Use the [sc.exe](https://technet.microsoft.com/library/bb490995) command-line tool to create the service (`sc create <SERVICE_NAME> binPath= "<PATH_TO_SERVICE_EXECUTABLE>"`). The `binPath` value is the path to the app's executable, which includes the executable file name. **The space between the equal sign and the quote character that starts the path is required.**
+1. Use the [sc.exe](https://technet.microsoft.com/library/bb490995) command-line tool to create the service. The `binPath` value is the path to the app's executable, which includes the executable file name. **The space between the equal sign and the quote character at the start of the path is required.**
 
-   For the sample app and command that follows, the service is:
+   ```console
+   sc create <SERVICE_NAME> binPath= "<PATH_TO_SERVICE_EXECUTABLE>"
+   ```
+
+   For a service published in the project folder, use the path to the *publish* folder to create the service. In the following example, the service is:
 
    * Named **MyService**.
-   * Published to *c:\\svc* folder.
-   * Has an app executable named *AspNetCoreService.exe*.
+   * Created from a project that sets the target framework to `netcoreapp2.1`.
```

Comment:
Does the `netcoreapp2.1` have to be used? Is there a way to future-proof this text?